README.md

```diff
@@ -71,5 +80,5 @@ jobs:
         with:
           WAKATIME_API_KEY: ${{ secrets.WAKATIME_API_KEY }}
           GH_TOKEN: ${{ secrets.GH_TOKEN}}
-          USERNAME: <username> # optional, it will automaticially use the username that executing the workflow
+          USERNAME: <username> # optional, it will automatically use the username that's executing the workflow
```

Comment:
```suggestion
          USERNAME: <username> # optional, it will automatically use the username of the owner of the repository that's executing the workflow
```

Reply:
Made requested changes.